src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.civil.controllers.cases;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.civil.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.CoreCaseDataService;
+
+@Api
+@Slf4j
+@RestController
+@AllArgsConstructor
+@RequestMapping(
+    path = "/cases",
+    produces = MediaType.APPLICATION_JSON_VALUE
+)
+public class CasesController {
+
+    private final CoreCaseDataService coreCaseDataService;
+    private final CaseDetailsConverter caseDetailsConverter;
+
+    @GetMapping(path = {
+        "/{caseId}",
+    })
+    @ApiOperation("get case by id from CCD")
```

Comment:
Is it case Id / case ref --> We have two differentiators for a claim. 
Case ID refers to 16 digit number where case ref refers to "27MC678" something like this.
We need to make sure what is being passed from UI to this call.

Reply:
I think it should be case ID as the CCD API endpoint for retrieving case data expects a case ID

Reply:
In adicional of  Taoufik's comment In the CoreCaseDataAPI is 
            @PathVariable("cid") String caseId